DancingGoat/Styles/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.css
+*.css.map
```

Comment:
I think it should be in the root `.gitignore` (mentioned in https://github.com/Kentico/kontent-sample-app-net/pull/123#discussion_r639457467 already)